src/test/resources/liquibase/harness/change/changelogs/databricks/alterCluster.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please add json and yaml changelogs, even if IT tests don't pick them by default, they can be run locally by adding -`DinputFormat=yaml`. Also, they are good references for manual testing.